tests/data/wpdb.php

```diff
@@ -14,8 +14,19 @@
 assertType('array<int, mixed>|void|null', wpdb::get_row(null, 'ARRAY_N'));
 
 // wpdb::get_results()
-assertType('stdClass|null', wpdb::get_results());
-assertType('stdClass|null', wpdb::get_results(null, 'OBJECT'));
-assertType('array<string, stdClass>|null', wpdb::get_results(null, 'OBJECT_K'));
-assertType('array<string, mixed>|null', wpdb::get_results(null, 'ARRAY_A'));
-assertType('array<int, mixed>|null', wpdb::get_results(null, 'ARRAY_N'));
+    /*
+     *                       Any of ARRAY_A | ARRAY_N | OBJECT | OBJECT_K constants.
+	 *                       ARRAY_A | ARRAY_N | OBJECT: return an array of rows indexed from 0 by SQL result row number.
+     *
+     *                       ARRAY_A: Each row is an associative array (column => value, ...),
+     *                       ARRAY_N: a numerically indexed array (0 => value, ...),
+	 *                       OBJECT: an object ( ->column = value ), respectively.
+     *
+     *                       With OBJECT_K: return an associative array of row objects keyed by the value
+	 *                       of each row's first column's value.
+     */
+assertType('array<int, array<string, mixed>>|null', wpdb::get_results(null, 'ARRAY_A'));
```

Comment:
```suggestion
assertType('array<int, array<mixed>>|null', wpdb::get_results(null, 'ARRAY_A'));
```

I would get rid of the `string` key here and let it fall back to `(int|string)` for the same reasons as as described in the other comment. WDYT?

Reply:
Please open a Martin-PR!